rpcs3/Emu/Cell/lv2/sys_btsetting.h

```diff
@@ -0,0 +1,8 @@
+﻿#pragma once
+
+#include "stdafx.h"
```

Comment:
I think this is not needed

Reply:
It's also undesirable to add a pch to a header file.

Reply:
Probably meant to include Types.h or something, not a big problem.